README.md

```diff
@@ -73,9 +73,9 @@ await wallet.signOut();
 
 ```typescript
 import { providers } from "ethers";
-import { init } from "@textile/eth-storage";
+import { init, requestSignIn } from "@textile/eth-storage";
 
-await window.ethereum.enable();
+await requestSignIn();
```

Comment:
though i like the abstraction, what it gains in simplicity i wonder if we are losing in interop/familiarity. as in, interacting with the ethereum object is a standard process in documentation for any dapp building. here, we have to explain something new/different than every other tutorial they might read.

Reply:
actually, I totally agree with you! thanks for flagging that. in hindsight, I think you're right. I'm going to leave the api available, but I'm going to revert the docs to use the more "standard" approach

Reply:
sweet